multiapi-engine/src/main/java/com/sngular/api/generator/plugin/asyncapi/AsyncApiGenerator.java

```diff
@@ -179,24 +202,22 @@ private void checkRequiredOrCombinatorExists(final SchemaObject schema, final bo
   private Map<String, JsonNode> getAllSchemas(final Path ymlParentPath, final JsonNode node) {
     final Map<String, JsonNode> totalSchemas = new HashMap<>();
     final List<JsonNode> referenceList = node.findValues(REF);
-    referenceList.forEach(reference ->
-        processReference(node, reference, ymlParentPath, totalSchemas, referenceList)
-    );
+    referenceList.forEach(reference -> processReference(node, reference, ymlParentPath, totalSchemas, referenceList));
     final List<JsonNode> messagesList = node.findValues(MESSAGES);
     final List<JsonNode> schemasList = node.findValues(SCHEMAS);
-    schemasList.forEach(schema -> schema.fields().forEachRemaining(fieldSchema -> totalSchemas.putIfAbsent((SCHEMAS + SLASH + fieldSchema.getKey()).toUpperCase(),
-                                                                   fieldSchema.getValue())));
+    schemasList.forEach(
+        schema -> schema.fields().forEachRemaining(fieldSchema -> totalSchemas.putIfAbsent((SCHEMAS + SLASH + fieldSchema.getKey()).toUpperCase(), fieldSchema.getValue())));
```

Comment:
Due to the code `SCHEMAS +SLASH +fieldSchema.getKey()` is repeated, at least three times (L. 209, 215 and 217), may be it should be a good idea to extract its value to a constant to avoid repeating the same code.